back/src/Application/DeletePost/DeletePostHandler.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\DeletePost;
+
+use App\Application\GetPosts\GetPostByIdInterface;
+use App\Domain\Exception\PostNotFoundException;
+
+class DeletePostHandler
+{
+    public function __construct(
+        private readonly DeletePostInterface $deletePost,
+        private readonly GetPostByIdInterface $getPostByIdQuery,
+    ) {
+    }
+
+    /**
+     * @throws PostNotFoundException
+     */
+    public function handle(DeletePostCommand $command): void
+    {
+        if (0 == $command->id) {
```

Comment:
Why 0 ? What if your id is equl to 0 ? You just want to check if the value is defined. You should also prefer `===` instead of `==` to check type and value
```suggestion
        if (null === $command->id) {
```